boards/px4/fmu-v2/init/rc.board_extras

```diff
@@ -0,0 +1,35 @@
+#!nsh
+#
+# PX4 FMUv2 specific board init
+#
+#------------------------------------------------------------------------------
+#
+# UART mapping on FMUv2:
```

Comment:
I'd keep the mapping only in one file.

Reply:
Should be done, but let's do a pass to review them all.